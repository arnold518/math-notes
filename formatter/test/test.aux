\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\@writefile{toc}{\contentsline {part}{I\hspace  {1em}SDF}{1}{part.1}\protected@file@percent }
\@writefile{toc}{\contentsline {chapter}{\numberline {1}Introduction}{2}{chapter.1}\protected@file@percent }
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\newlabel{theorem:1.3}{{1}{2}{Introduction}{section*.1}{}}
\newlabel{concept:2.1}{{1}{2}{Introduction}{section*.2}{}}
\@writefile{toc}{\contentsline {chapter}{\numberline {2}Images and Lists}{3}{chapter.2}\protected@file@percent }
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{toc}{\contentsline {section}{\numberline {2.1}Lists in LaTeX}{3}{section.2.1}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {2.1.1}Unordered List}{3}{subsection.2.1.1}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {2.1.2}Ordered List}{3}{subsection.2.1.2}\protected@file@percent }
\@writefile{toc}{\contentsline {chapter}{\numberline {3}Code Blocks}{4}{chapter.3}\protected@file@percent }
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{toc}{\contentsline {section}{\numberline {3.1}Simple Code (Verbatim)}{4}{section.3.1}\protected@file@percent }
\@writefile{toc}{\contentsline {chapter}{\numberline {4}Optimization Problem}{5}{chapter.4}\protected@file@percent }
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{toc}{\contentsline {section}{\numberline {4.1}Definition of Optimization Problem}{5}{section.4.1}\protected@file@percent }
\newlabel{definition:1.1}{{4.1}{5}{Definition of Optimization Problem}{section*.3}{}}
\newlabel{definition:1.2}{{4.1}{5}{Definition of Optimization Problem}{section*.4}{}}
\newlabel{definition:1.3}{{4.1}{6}{Definition of Optimization Problem}{section*.5}{}}
\@writefile{toc}{\contentsline {section}{\numberline {4.2}Examples of Optimization Problem}{6}{section.4.2}\protected@file@percent }
\newlabel{example:1.4}{{4.2}{6}{Examples of Optimization Problem}{section*.6}{}}
\newlabel{example:1.5}{{4.2}{6}{Examples of Optimization Problem}{section*.7}{}}
\newlabel{concept:1.6}{{4.2}{7}{Examples of Optimization Problem}{section*.8}{}}
\@writefile{toc}{\contentsline {section}{\numberline {4.3}Local and Global Minimum}{7}{section.4.3}\protected@file@percent }
\newlabel{definition:1.7}{{4.3}{7}{Local and Global Minimum}{section*.9}{}}
\@writefile{toc}{\contentsline {chapter}{\numberline {5}Basics of Monte Carlo}{9}{chapter.5}\protected@file@percent }
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{toc}{\contentsline {section}{\numberline {5.1}Monte Carlo Estimation}{9}{section.5.1}\protected@file@percent }
\newlabel{definition:13.1}{{5.1}{9}{Monte Carlo Estimation}{section*.10}{}}
\newlabel{concept:13.2}{{5.1}{9}{Monte Carlo Estimation}{section*.11}{}}
\newlabel{definition:13.3}{{5.1}{10}{Monte Carlo Estimation}{section*.12}{}}
\newlabel{concept:13.4}{{5.1}{10}{Monte Carlo Estimation}{section*.13}{}}
\@writefile{toc}{\contentsline {section}{\numberline {5.2}Importance Sampling}{11}{section.5.2}\protected@file@percent }
\newlabel{definition:13.5}{{5.2}{11}{Importance Sampling}{section*.14}{}}
\newlabel{example:13.6}{{5.2}{11}{Importance Sampling}{section*.15}{}}
\newlabel{concept:13.7}{{5.2}{11}{Importance Sampling}{section*.16}{}}
\newlabel{concept:13.8}{{5.2}{12}{Importance Sampling}{section*.17}{}}
\@writefile{toc}{\contentsline {section}{\numberline {5.3}Log-Derivative Trick}{12}{section.5.3}\protected@file@percent }
\newlabel{definition:13.9}{{5.3}{12}{Log-Derivative Trick}{section*.18}{}}
\newlabel{example:13.10}{{5.3}{13}{Log-Derivative Trick}{section*.19}{}}
\@writefile{toc}{\contentsline {section}{\numberline {5.4}Reparameterization Trick}{14}{section.5.4}\protected@file@percent }
\newlabel{definition:13.11}{{5.4}{14}{Reparameterization Trick}{section*.20}{}}
\newlabel{example:13.12}{{5.4}{14}{Reparameterization Trick}{section*.21}{}}
\newlabel{example:13.13}{{5.4}{15}{Reparameterization Trick}{section*.22}{}}
\gdef \@abspage@last{16}
