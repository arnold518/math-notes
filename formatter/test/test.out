\BOOKMARK [-1][-]{part.1}{\376\377\000I\000\040\000S\000D\000F}{}% 1
\BOOKMARK [0][-]{chapter.1}{\376\377\000I\000n\000t\000r\000o\000d\000u\000c\000t\000i\000o\000n}{part.1}% 2
\BOOKMARK [0][-]{chapter.2}{\376\377\000I\000m\000a\000g\000e\000s\000\040\000a\000n\000d\000\040\000L\000i\000s\000t\000s}{part.1}% 3
\BOOKMARK [1][-]{section.2.1}{\376\377\000L\000i\000s\000t\000s\000\040\000i\000n\000\040\000L\000a\000T\000e\000X}{chapter.2}% 4
\BOOKMARK [2][-]{subsection.2.1.1}{\376\377\000U\000n\000o\000r\000d\000e\000r\000e\000d\000\040\000L\000i\000s\000t}{section.2.1}% 5
\BOOKMARK [2][-]{subsection.2.1.2}{\376\377\000O\000r\000d\000e\000r\000e\000d\000\040\000L\000i\000s\000t}{section.2.1}% 6
\BOOKMARK [0][-]{chapter.3}{\376\377\000C\000o\000d\000e\000\040\000B\000l\000o\000c\000k\000s}{part.1}% 7
\BOOKMARK [1][-]{section.3.1}{\376\377\000S\000i\000m\000p\000l\000e\000\040\000C\000o\000d\000e\000\040\000\050\000V\000e\000r\000b\000a\000t\000i\000m\000\051}{chapter.3}% 8
\BOOKMARK [0][-]{chapter.4}{\376\377\000O\000p\000t\000i\000m\000i\000z\000a\000t\000i\000o\000n\000\040\000P\000r\000o\000b\000l\000e\000m}{part.1}% 9
\BOOKMARK [1][-]{section.4.1}{\376\377\000D\000e\000f\000i\000n\000i\000t\000i\000o\000n\000\040\000o\000f\000\040\000O\000p\000t\000i\000m\000i\000z\000a\000t\000i\000o\000n\000\040\000P\000r\000o\000b\000l\000e\000m}{chapter.4}% 10
\BOOKMARK [1][-]{section.4.2}{\376\377\000E\000x\000a\000m\000p\000l\000e\000s\000\040\000o\000f\000\040\000O\000p\000t\000i\000m\000i\000z\000a\000t\000i\000o\000n\000\040\000P\000r\000o\000b\000l\000e\000m}{chapter.4}% 11
\BOOKMARK [1][-]{section.4.3}{\376\377\000L\000o\000c\000a\000l\000\040\000a\000n\000d\000\040\000G\000l\000o\000b\000a\000l\000\040\000M\000i\000n\000i\000m\000u\000m}{chapter.4}% 12
\BOOKMARK [0][-]{chapter.5}{\376\377\000B\000a\000s\000i\000c\000s\000\040\000o\000f\000\040\000M\000o\000n\000t\000e\000\040\000C\000a\000r\000l\000o}{part.1}% 13
\BOOKMARK [1][-]{section.5.1}{\376\377\000M\000o\000n\000t\000e\000\040\000C\000a\000r\000l\000o\000\040\000E\000s\000t\000i\000m\000a\000t\000i\000o\000n}{chapter.5}% 14
\BOOKMARK [1][-]{section.5.2}{\376\377\000I\000m\000p\000o\000r\000t\000a\000n\000c\000e\000\040\000S\000a\000m\000p\000l\000i\000n\000g}{chapter.5}% 15
\BOOKMARK [1][-]{section.5.3}{\376\377\000L\000o\000g\000-\000D\000e\000r\000i\000v\000a\000t\000i\000v\000e\000\040\000T\000r\000i\000c\000k}{chapter.5}% 16
\BOOKMARK [1][-]{section.5.4}{\376\377\000R\000e\000p\000a\000r\000a\000m\000e\000t\000e\000r\000i\000z\000a\000t\000i\000o\000n\000\040\000T\000r\000i\000c\000k}{chapter.5}% 17
